libraries/Microsoft.Bot.Builder/BotFrameworkAdapter.cs

```diff
@@ -784,7 +784,8 @@ public virtual async Task CreateConversationAsync(string channelId, string servi
             eventActivity.ChannelId = channelId;
             eventActivity.ServiceUrl = serviceUrl;
             eventActivity.Id = result.ActivityId ?? Guid.NewGuid().ToString("n");
-            eventActivity.Conversation = new ConversationAccount(id: result.Id);
+            eventActivity.Conversation = new ConversationAccount(id: result.Id, tenantId: conversationParameters.TenantId);
+            eventActivity.ChannelData = conversationParameters.ChannelData;
```

Comment:
I feel like this is a valid change. 

Setting this event's ChannelData to the conversationParameters.ChannelData will definitely not hurt anything while on the TeamsChannel, I doubt there will be unknown consequences on other channels, but i suppose it is possible.??